app/models/votable_field.rb

```diff
@@ -0,0 +1,18 @@
+class VotableField < ActiveRecord::Base
+  belongs_to :conference
+  validates :title, :votable_type, presence: true
+
+  # ratyrate does not allow criterias to have spaces in them
+  validate :no_spaces_in_title
```

Comment:
Another solution could be to replace spaces with another character like underscore ( _ ) and convert them back to spaces while rendering them in proposals#show . But this could be a problem if the user actually defines a field that has a underscore.